pkg/report/timeseries.go

```diff
@@ -128,9 +128,9 @@ func (ts TimeSeries) String() string {
 	for i := range ts {
 		row := []string{
 			fmt.Sprintf("%d", ts[i].Timestamp),
-			fmt.Sprintf("%s", ts[i].MinLatency),
-			fmt.Sprintf("%s", ts[i].AvgLatency),
-			fmt.Sprintf("%s", ts[i].MaxLatency),
+			fmt.Sprintf("%v", ts[i].MinLatency),
```

Comment:
I tried it with string(whatever) but that got binary output. whatever.String() turns out to be OK.